tools/ExtensionsMetadataGenerator/test/ExtensionsMetadataGeneratorTests/ExistingRuntimeAssemblies.txt

```diff
@@ -83,7 +82,6 @@ Microsoft.Azure.WebJobs.Extensions.dll
 Microsoft.Azure.WebJobs.Extensions.Http.dll
 Microsoft.Azure.WebJobs.Host.dll
 Microsoft.Azure.WebJobs.Logging.dll
-Microsoft.Azure.WebJobs.Logging.ApplicationInsights.dll
```

Comment:
Because of this, we'll need a 4.0.0-preview1 ExtensionsMetadataGenerator build, right? I can follow-up with getting that build ready in a separate PR.

Reply:
Correct, yeah. This is why for now you have to add the app insights dlls to the "don't clean" list of your test app